data/hai outfits.txt

```diff
@@ -247,7 +247,7 @@ outfit "Hai Williwaw Cooling"
 	"mass" 8
 	"outfit space" -8
 	"cooling" 9.5
-	description "While Hai technology is fairly heat efficient, occasionally they outfit their ships such that they sporadically overheat and shut down. When that happens, they install these."
+	description "While Hai technology is fairly heat-efficient, occasionally they outfit their ships such that they sporadically overheat and shut down. When that happens, they install these."
```

Comment:
The two words are acting as an adjective, but there is no noun directly following it. "Heat-efficient Hai technology" is correct, whereas "Hai technology is heat-efficient" does not.

This is correct usage: "Heat-efficient technology is heat efficient". The phrase "heat efficient" is still acting as an adjective to technology, but there is no noun directly following it.

The link doesn't seem to be working, so if you're not seeing it, copy and paste the url.